stdlib/public/Python/Python.swift

```diff
@@ -1227,29 +1232,21 @@ extension PythonObject : Hashable {
   }
 }
 
-extension PythonObject : MutableCollection {
```

Comment:
Is it necessary to delete this? I think your change should be purely additive.

Reply:
Interesting, without removing this change It is falling back to the same iteration. I might be doing something wrong here.

Reply:
In think you would need to implement:
public func makeIterator() -> PythonObject { return Python.iter(self) }
as part of a conformance to Sequence?

Reply:
Adding Sequence conformance works. Thanks 👍